src/plugins/orangehrmDashboardPlugin/Service/ChartService.php

```diff
@@ -71,4 +72,47 @@ static function (SubunitCountPair $x, SubunitCountPair $y) {
             $limit
         );
     }
+
+    /**
+     * @param array $locationEmployeeCounts
+     * @return int
+     */
+    public function getLocationUnassignedEmployeeCount(array $locationEmployeeCounts): int
+    {
+        $totalActiveEmployee = $this->chartDao->getTotalActiveEmployeeCount();
+
+        $assignedEmployeeCount = 0;
+        foreach ($locationEmployeeCounts as $locationEmployeeCount) {
+            $assignedEmployeeCount += $locationEmployeeCount->getEmployeeCount();
+        }
+        return $totalActiveEmployee-$assignedEmployeeCount;
+    }
+
+    /**
+     * @param int $limit
+     * @return EmployeeDistributionByLocation
+     */
+    public function getEmployeeDistributionByLocation(int $limit=8): EmployeeDistributionByLocation
+    {
+        $locationEmployeeCount = $this->getChartDao()->getEmployeeDistributionByLocation();
+        $unassignedEmployeeCount =  $this->getLocationUnassignedEmployeeCount($locationEmployeeCount);
```

Comment:
can't we calculate `$unassignedEmployeeCount` inside this method, rather than execute another for loop in `getLocationUnassignedEmployeeCount`

Reply:
Add to https://github.com/orangehrm/orangehrm/pull/1440